src/App.tsx

```diff
@@ -1,19 +1,176 @@
 import React from 'react';
-import './App.css';
+import classNames from 'classnames';
+import { TodoList } from './components/TodoList/TodoList';
+
+import { User } from './types/User';
+import { TodoPrepared } from './types/TodoPrepared';
+
+import './App.scss';
 
 import users from './api/users';
+import todos from './api/todos';
+
+const preparedTodos = todos.map(todo => {
+  const user = users.find(person => person.id === todo.userId);
 
-const App: React.FC = () => {
-  return (
-    <div className="App">
-      <h1>Add todo form</h1>
-
-      <p>
-        <span>Users: </span>
-        {users.length}
-      </p>
-    </div>
-  );
+  return {
+    user: user || null,
+    ...todo,
+  };
+});
+
+type State = {
+  usersCopy: User[],
+  todosCopy: TodoPrepared[],
+  todoTitle: string,
+  selectedUserId: number,
+  selectError: boolean,
```

Comment:
by name, I can’t understand that value is boolean
```suggestion
  isSelectUser: boolean,
```